db/migrations/20230227122704-ss58-prefix.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
```

Comment:
I guess because we use a modern version of node.js we don't need that anymore

Reply:
It's autogenerated script by running `yarn workspace backend db:migrate:create`, so each transaction starts with
I think it's better to follow existing standards provided by the `db-migrate-pg` library so I didn't change this part of code. 
```
'use strict';

var dbm;
var type;
var seed;
var keyring = require('@polkadot/keyring');
var Promise;

/**
 * We receive the dbmigrate dependency from dbmigrate initially.
 * This enables us to not have to rely on NODE_PATH.
 */
exports.setup = function (options, seedLink) {
    dbm = options.dbmigrate;
    type = dbm.dataType;
    seed = seedLink;
    Promise = options.Promise;
};
```